discovery/syncer_test.go

```diff
@@ -295,19 +296,20 @@ func TestGossipSyncerFilterGossipMsgsAllInMemory(t *testing.T) {
 	// Before we send off the query, we'll ensure we send the missing
 	// channel update for that final ann. It will be below the horizon, so
 	// shouldn't be sent anyway.
+	errCh := make(chan error)
```

Comment:
buffer with 1?

Reply:
ok

Reply:
Doesn't really need to be buffered since the send happens in another goroutine, but fine as is.